tests/unit_tests/test_heron.py

```diff
@@ -8,7 +8,36 @@
 sys.path.append(FORCE_LOC)
 from FORCE.src.heron import create_componentsets_in_HERON
 
-class TestMinimalInput(unittest.TestCase):
+class HERONTestCase(unittest.TestCase):
+  
+  def check_reference_price(self, cashflow, correct_value, correct_content_length=1):
```

Comment:
would you be able to add docstrings to these new methods? For reference: https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#function-and-class-documentation-strings

Reply:
No problem.